### ISSUE TITLE
`minishift delete` should ask for confirmation before deletion

### ISSUE THREAD

**Issue by gbraad** [minishift/minishift#405]
Performing a delete should be an option that asks for confirmation

## Scenario

```gherkin
Given a running minishift instance
 When I want to undefine the instance using `minishift delete`
  And when the user answers with 'y'
 Then the images for the instance are deleted
  And the minishift instance should be unavailable with `sudo virsh list`
```

## Expected result
```
$ minishift delete
Do you really want to remove the minishift instance? [y/N]
```
where *no* is the default option when entered.

## Actual result
Currently, the action does not ask for a confirmation and the instance gets deleted
```
$ minishift delete
Deleting the Minishift VM...
^C
```
Note: breaking out will not help you

**Comment by budhrg** [comment]
:+1:

**Comment by hferentschik** [comment]
If we go down this road, we need also to add a '--force' flag.

**Comment by LalatenduMohanty** [comment]
+1 for asking for confirmation and `--force` flag.

**Comment by praveenkumar** [comment]
@hferentschik Can we add a milestone to it, seems like a trivial to do.

**Comment by hferentschik** [comment]
Well, I am not so sure. If we do it, we also need to add a --force, since otherwise scripting and testing becomes a pain. But let's discuss.

**Comment by LalatenduMohanty** [comment]
I have fixed this as part of #126

**Comment by budhrg** [comment]
Resolved via https://github.com/minishift/minishift/pull/1380
